### ISSUE TITLE
Hide `bus=no` stops

### ISSUE THREAD

**Issue by Dimitar5555** [Zaczero/osm-relatify#39]
Currently when a stop is tagged with `bus=no`, it's still shown in Relatify. It may sound like a troll tag but it is used in situation where other vehicles (like trolleybuses) have their own road or when trolleybuses serve all stops while buses run express services.

![image](https://github.com/Zaczero/osm-relatify/assets/19364673/88073355-ba87-4141-99d7-b3803e197532)

The sign that is closer to the camera is the bus stop, the sign in the back is for the trolleybus.
![image](https://github.com/Zaczero/osm-relatify/assets/19364673/a47ca979-7e22-4bc5-a450-d4fb9aada30e)

**Comment by Hidooo** [comment]
`bus=no` is also used for bus stops without any buses stopping, such as when lines are rerouted but the infrastructure is kept in place for future line rerouting. In such cases, the "inactive" bus stops still need to be shown to be able to add new lines.

**Comment by Zaczero** [comment]
> `bus=no` is also used for bus stops without any buses stopping, such as when lines are rerouted but the infrastructure is kept in place for future line rerouting. In such cases, the "inactive" bus stops still need to be shown to be able to add new lines.

Oh, considering the new information, this does complicate things a bit. If we're only discussing a visual feature, then it seems like it's not worth pursuing.

**Comment by Dimitar5555** [comment]
> `bus=no` is also used for bus stops without any buses stopping, such as when lines are rerouted but the infrastructure is kept in place for future line rerouting. In such cases, the "inactive" bus stops still need to be shown to be able to add new lines.

That sounds like a perfect use case for `disused:highway=bus_stop`.

**Comment by Hidooo** [comment]
> That sounds like a perfect use case for `disused:highway=bus_stop`.

Definitely, but then that needs to be handled by relatify if it currently isn't.
